### ISSUE TITLE
app method chaining doesn't work

### ISSUE THREAD

**Issue by maxpain** [dimdenGD/ultimate-express#64]
Hello. I'm migrating from legacy `express` to `ultimate-express` and have to change this code:

```typescript
app
	.use(express.json())
	.use(hasuraHeadersMiddleware)
	.use(cookieParser())
	.use(sessionMiddleware)
	.use(routes)
	.get('/healthz', (req, res) => {
		res.status(200)
	})
```

to this:
```typescript
app.use(express.json())
app.use(hasuraHeadersMiddleware)
app.use(cookieParser())
app.use(sessionMiddleware)
app.use(routes)
app.get('/healthz', (req, res) => {
	res.status(200)
})
```

Because `ultimate-express` doesn't support `app` method chaining.
```
TypeError: Cannot read properties of undefined (reading 'use')
    at <anonymous> (/Users/maxpain/dev/fastcup/backend/auth/src/index.ts:19:21)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
```

**Comment by nigrosimone** [comment]
fixed by https://github.com/dimdenGD/ultimate-express/pull/65

**Comment by dimdenGD** [comment]
Released 1.3.12 on NPM with fix
